src/pages/logga-ut.tsx

```diff
@@ -0,0 +1,20 @@
+import { api } from "@/utils/api";
+import { useRouter } from "next/router";
+import { useEffect } from "react";
+
+export default function Logout() {
+  const router = useRouter();
+  const trpc = api.useContext();
+  const logout = api.account.logout.useMutation();
+
+  useEffect(() => {
+    trpc.account.invalidate();
+    router.replace("/");
+  }, [logout.isSuccess]);
+
```

Comment:
We should add a loading gif like on the exhibitor page so that the footer doesn't occupy the whole page.

Reply:
Fixed it by making the log out button log the user out directly instead of having a "logging out" page